0.1.0/rockcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: metrics-proxy
+summary: Metrics Proxy for Prometheus scraping
+description: A metrics proxy server that aggregates Prometheus metrics from Kubernetes pods based on kubernetes service discovery annotations.
+license: Apache-2.0
+
+version: "0.1.0"
+base: "ubuntu@22.04"
+platforms:
+  amd64:
+
+services:
+  metrics-proxy:
+    override: replace
+    startup: enabled
+    command: metrics-proxy --labels="foo=bar"
```

Comment:
`--labels="X=Y"` here will be set dynamically at runtime.  Should we change this to use arguments instead?  For example, like [here](https://github.com/canonical/seldonio-rocks/issues/83#issuecomment-1896270699)

Reply:
Yeah defaulting to foo bar seems bad here. Let's at least pretend someone might use this outside of juju.